db/seeds/support/data_migrator.rb

```diff
@@ -94,18 +105,27 @@ def role_attrs_from_function(row, person)
 
       return unless role_type && section
 
-      [
+      section_attrs = [
```

Comment:
uiuiui das ist mir ein bisschen zu wirrwarr-ig... Denke es macht Sinn das in eine eigene Methode zu extrahieren die nur die attrs für Freimitglieder returned :)